kse/src/org/kse/gui/dialogs/extensions/resources_fr.properties

```diff
@@ -2,28 +2,28 @@
 
 
 
-DViewExtensions.ViewCert.Title=D�tails du certificat pour {0}
-DViewExtensions.ViewCrl.Title=D�tails de la liste de r�vocation pour {0}
+DViewExtensions.ViewCert.Title=D�tails du certificat pour {0}
```

Comment:
The encoding of the french resource bundle is messed up. Java (up to version 8) requires resource bundles to be encoded as ISO 8859-1.

Reply:
Ok. I've changed encoding and fixed the texts in french bundle.